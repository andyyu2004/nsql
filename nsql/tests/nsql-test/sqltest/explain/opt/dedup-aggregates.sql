EXPLAIN
SELECT
    SUM(a),
    SUM(a),
    SUM(a)
FROM (
    VALUES (1),
        (2),
        (3)) AS t (a)
GROUP BY
    a
ORDER BY
    SUM(a)
