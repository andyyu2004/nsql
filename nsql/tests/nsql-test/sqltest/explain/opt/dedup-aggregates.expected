 projection (:project @0 :return, :project @1 :return, :project @2 :return)                          
   order by (:project @3 :return)                                                                    
     projection (:project @1 :return, :project @1 :return, :project @1 :return, :project @1 :return) 
       hash aggregate (sum(:project @0 :return)) by :project @0 :return                              
         scan 3 values                                                                               
                                                                                                     